WordPress/src/main/java/org/wordpress/android/ui/reader/viewmodels/ReaderViewModel.kt

```diff
@@ -99,8 +102,14 @@ class ReaderViewModel @Inject constructor(
         return now - lastUpdated > UPDATE_TAGS_THRESHOLD
     }
 
+    fun emptyStateTabChange(tag: ReaderTag) {
+        onTagChanged(tag)
```

Comment:
Do we need to manually call `onTagChanged`? I'd expect when the tab is changed it'll invoke the `onPageChangeCallback` and the tag gets eventually changed. I haven't tested it, so I might be wrong. Let me know what you think ;).

Reply:
> Do we need to manually call onTagChanged?

Nope, the callback will take care of it. Going to remove it.